frontend/src/lib/components/AuthorizedUrlList/authorizedUrlListLogic.ts

```diff
@@ -56,6 +68,10 @@ export const validateProposedUrl = (
         return 'Please enter a valid URL'
     }
 
+    if (hasPortWildcard(proposedUrl)) {
```

Comment:
can we add some test cases to https://github.com/PostHog/posthog/blob/b5fbf6d51e859615b7956e0d63eac7be12ba1167/frontend/src/lib/components/AuthorizedUrlList/authorizedUrlListLogic.test.ts#L72 so I don't have to think about the regex 🤣 

I guess do we support e.g. `https://*.example.com:*001`

Reply:
> I guess do we support e.g. https://*.example.com:*001

Where do you see that? I don't see it in any of the test cases?

Reply:
just as an example testcase :)

Reply:
or asking if we support it...